python/ql/lib/semmle/python/security/OverlyLargeRangeQuery.qll

```diff
@@ -238,8 +238,13 @@ module RangePrinter {
 
 /** Gets a char range that is overly large because of `reason`. */
 RegExpCharacterRange getABadRange(string reason, int priority) {
+  result instanceof OverlyWideRange and
```

Comment:
I'm surprised that this `instanceof` check is needed. Isn't this implied by the inline cast on line 243? :thinking:

Reply:
You're right, that check is not needed, but I'm going this merge this PR as is anyway.  
The cast was needed in one of the first versions of my fix, but it's not needed with the version I ended up with.  

Maybe a QL-for-QL query in the style of `ql/superfluous-exists`?